tutorials/scripting/creating_script_templates.rst

```diff
@@ -203,7 +203,9 @@ Base placeholders
 +==========================+====================================================+
 | ``_BINDINGS_NAMESPACE_`` | The name of the Godot namespace (used in C# only). |
 +--------------------------+----------------------------------------------------+
-| ``_CLASS_``              | The name of the new class.                         |
+| ``_CLASS_``              | The name of the new class as ``PascalCase``.       |
```

Comment:
This substitution contains the original class name, not necessarily in `PascalCase`. Since the user may not follow the recommended style.